File: create_comments_table.py
import requests

BASE_URL = "https://eventhub-nextjs-three.vercel.app"

def create_comments_table():
    print("Creating Comments Table")
    print("=" * 25)
    
    response = requests.post(f"{BASE_URL}/api/create-comments-table")
    print(f"Status: {response.status_code}")
    
    if response.status_code == 200:
        result = response.json()
        print("SUCCESS - Comments table created!")
        print(f"Message: {result['message']}")
        print(f"Current count: {result['count']}")
    else:
        print("FAILED to create comments table")
        print(f"Error: {response.text}")

if __name__ == "__main__":
    create_comments_table()

File: pages/api/create-comments-table.js
import { sql } from '@vercel/postgres';

export default async function handler(req, res) {
  if (req.method === 'POST') {
    try {
      console.log('Creating event_comments table...');
      
      // Create comments table separately
      await sql`
        CREATE TABLE IF NOT EXISTS event_comments (
          id SERIAL PRIMARY KEY,
          event_id INTEGER REFERENCES events(id) ON DELETE CASCADE,
          student_id INTEGER REFERENCES students(id) ON DELETE CASCADE,
          comment TEXT NOT NULL,
          created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
        )
      `;
      
      console.log('event_comments table created successfully');
      
      // Test if table exists by querying it
      const result = await sql`SELECT COUNT(*) FROM event_comments`;
      console.log('Table test query successful:', result.rows[0]);
      
      res.status(200).json({ 
        message: 'Comments table created successfully',
        count: result.rows[0].count 
      });
    } catch (error) {
      console.error('Error creating comments table:', error);
      res.status(500).json({ 
        message: 'Failed to create comments table', 
        error: error.message 
      });
    }
  } else {
    res.status(405).json({ message: 'Method not allowed' });
  }
}